catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/opensearch/query/filter/BBoxSpatialFilter.java

```diff
@@ -54,14 +61,29 @@ public BBoxSpatialFilter(String bbox) {
    * @return the wkt String
    */
   private String createWKT() {
-    StringBuilder wktBuilder = new StringBuilder("POLYGON((");
-    wktBuilder.append(minX + " " + minY);
-    wktBuilder.append("," + minX + " " + maxY);
-    wktBuilder.append("," + maxX + " " + maxY);
-    wktBuilder.append("," + maxX + " " + minY);
-    wktBuilder.append("," + minX + " " + minY);
-    wktBuilder.append("))");
+    String wktBuilder =
```

Comment:
It was a static analysis finding to use string concatenation instead of StringBuilder. For readability, I replaced this with 
```java
String.format("POLYGON((%1$s %2$s,%1$s %4$s,%3$s %4$s,%3$s %2$s,%1$s %2$s))", minX, minY, maxX, maxY)
```